subprojects/logging/logging.gradle

```diff
@@ -18,6 +18,7 @@ dependencies {
     implementation libraries.jul_to_slf4j
     implementation libraries.ant
     implementation libraries.commons_lang
+    implementation libraries.commons_io
```

Comment:
Unfortunately, using guava would make the TAPI jar larger than using commons io because TAPI only contains classes that are referenced. Now I use pure JDK implementation instead of commons io.